packages/vertica-nodejs/script/create-test-tables.js

```diff
@@ -68,7 +68,7 @@ con.connect((err) => {
       console.log('Created table person')
       console.log('Filling it with people')
 
-      let q = people.map((person) => `INSERT INTO person (name, age) VALUES ('${person.name}', ${person.age})`).join(';')
+      let q = people.map((person) => `INSERT INTO person (name, age) VALUES ('${person.name}', ${person.age})`).join(';') + ";COMMIT;"
```

Comment:
Will we be removing this file entirely in favour of the local temp tables Danny suggested?

Reply:
Should be an eventual goal but would need to do it for a lot of tests so imo should be lower priority.